src/test/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/resource/ScheduleControllerIntTest.kt

```diff
@@ -119,6 +120,37 @@ class ScheduleControllerIntTest : IntegrationTestBase() {
       )
   }
 
+  @Test
+  @Sql(
+    "classpath:test_data/reset-all-data.sql",
+    "classpath:test_data/insert-offence-data-sexual-or-violent.sql",
+  )
+  fun `Get Sexual or Violent lists`() {
+    val result = webTestClient.get().uri("/schedule/sexual-or-violent-lists")
+      .headers(setAuthorisation())
+      .exchange()
+      .expectStatus().isOk
+      .expectBody(SexualOrViolentLists::class.java)
+      .returnResult().responseBody
+
+    val returnedLists = result ?: SexualOrViolentLists()
+    val sexualOffenceToScheduleCodes = returnedLists.sexual.map { it.code }.toList()
+    val violentOffenceToScheduleCodes = returnedLists.violent.map { it.code }.toList()
+
+    assertThat(sexualOffenceToScheduleCodes.size).isEqualTo(2)
```

Comment:
rather than all the different assertions, could we change this test to do an exact match ... e.g. `assertThat(result).isEqualTo`